lib/text_content/formatter.ex

```diff
@@ -42,14 +43,10 @@ defmodule Bonfire.Tag.TextContent.Formatter do
     end
   end
 
-  defp has_codeblock?(text) do
-    String.split(text, "`", parts: 3) == 3 || String.split(text, "```", parts: 3) == 3
-  end
-
-  defp linkify_opts(has_codeblock \\ false) do
+  defp linkify_opts() do
     Config.get(Bonfire.Tag.TextContent.Formatter, []) ++
       [
-        url_handler: if(has_codeblock, do: &nothing_handler/3, else: &url_handler/3),
```

Comment:
this was a temporary workaround, but without it we'd need to fix Linkify so that it doesn't turn transform links (or hashtags or mentions) that are inside a codeblock

Reply:
problem is that `has_codeblock` was wrong all along, I think. It's not working as intended, always returning false. I added a test before removing the code and after removing, it still behaves correctly. I believe linkify is taking care of it already, maybe in a more recent update, idk.